scripts/AEappleseedBumpControl.mel

```diff
@@ -1,31 +1,4 @@
 
-#
```

Comment:
Why removing the copyright notice?

Reply:
I'll update all MEL scripts soon, but the notice has to use ```//``` since MEL objects to the ```#``` character.